docker-compose.yml

```diff
@@ -1,29 +1,35 @@
-version: '0.1'
-
+version: '3.7'
 services:
   drone-server:
-    image: drone/drone:1
-    restart: always
+    container_name: drone-server
     environment:
       - DRONE_GITHUB_CLIENT_ID
       - DRONE_GITHUB_CLIENT_SECRET
       - DRONE_RPC_SECRET
       - DRONE_SERVER_HOST
       - DRONE_SERVER_PROTO
       - DRONE_TLS_AUTOCERT
+    image: drone/drone:1
     ports:
-      - "80:80"
-      - "443:443"
-  drone-runner:
-    image: drone/drone-runner-docker:1
+      - '80:80'
+      - '443:443'
     restart: always
     volumes:
-      - /var/run/docker.sock:/var/run/docker.sock
+      - drone-server-vol:/data
+  drone-runner:
+    container_name: drone-runner
     environment:
       - DRONE_RPC_PROTO
       - DRONE_RPC_HOST
       - DRONE_RPC_SECRET
       - DRONE_RUNNER_CAPACITY
       - DRONE_RUNNER_NAME
+    image: drone/drone-runner-docker:1
     ports:
-      - "3000:3000"
\ No newline at end of file
+      - '3000:3000'
+    restart: always
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+volumes:
```

Comment:
Added a volume for drone-server.  That way a new volume isn't created each time the container is removed and it remembers previous settings.